docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -25,6 +25,10 @@ The Plugins and Providers tabs display each plugin/provider on individual cards
 - a **Learn more** button for additional information, including detailed implementation instructions
 - a **Copy install command** button to copy the installation command to the local clipboard
 
+For any installed plugins and providers, the **Copy install command** button is replaced by an indicator that it is already installed:
+
+![Installed providers indicated](../assets/plugins/installed-providers.png)
```

Comment:
I think we could simplify that doc a bit 🤔 
How about:
- We replace the `marketplace-v4.png` screenshot with one that shows both "Copy install command" and "Installed" (either you use the new `installed-providers.png` screenshot, or you create a new one that shows the "Plugins" tab).
- We delete "For any installed plugins and providers, the **Copy install command** button is replaced by an indicator that it is already installed:" and instead:
   - either, add the new piece of information at the end of the last bullet point right above 👉  "a **Copy install command** button to copy the installation command to the local clipboard, replaced by an installation indicator when the plugin/provider is already installed" (or something like that)
   - or, mention that piece of information at the end of the "Installing Marketplace plugins and providers" section 👉  something like "Once installed, the cards of the plugin/provider displays an indicator that it is already installed, replacing the **Copy install command** button". And it can be in a callout if it looks better.

I think it would fit better the existing content. That addition (sentence + screenshot) seems a bit too much compared to the size/level of importance of that "Installed" indicator. What do you think? 🤓